### PR TITLE
Encapsulation (Matrix class added)

### PR DESCRIPTION
**Pull request by vbre**

### CODE REVIEW

File: Encapsulation/Valeriya/Money.cs [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            {
+                result = true;
+            }
+
+            return result;
+        }
+
+        public static bool operator> (Money operand1, Money operand2)
+        {
+            bool result = false;
```
Comment (jane-u) [inline Encapsulation/Valeriya/Money.cs]: one-line method is better in this case:
return operand1.kopeiky > operand2.kopeiky;

Comment (vbre) [inline Encapsulation/Valeriya/Money.cs]: All fixed

File: Miner/Valeriya/MinerGameFactory.cs [hunk 2]
```diff
@@ -65,9 +65,25 @@ internal void Test()
             }
         }
 
+<<<<<<< HEAD
```
Comment (jane-u) [inline Miner/Valeriya/MinerGameFactory.cs:68]: please remove artefacts of merging

Comment (vbre) [inline Miner/Valeriya/MinerGameFactory.cs:68]: It seems to be already removed as I had merged proper branch with master

**Pull request merged**